content/root/tools.textile

```diff
@@ -0,0 +1,83 @@
+    - React Hooks#react-hooks
+---
+
+This topic lists some developer resources and tools you can use to increase your productivity with Ably.
+
+h2(#vscode-plugin). Ably for Visual Studio Code
+
+Use the Ably for Visual Studio Code extension to manage your Ably apps directly in Visual Studio Code.
+
+The extension enables you to:
+
+* List all apps in your Ably account.
+** View API keys, queues and integration rules for each app.
+** Copy API keys to your clipboard.
+* Create an Ably app.
+
+"Find it here.":https://github.com/ably-labs/vscode-ably
+
+h2(#terraform-provider). Ably Terraform provider
+
+Terraform provider for Ably enables you to manage your Ably account using Infrastructure-as-Code (IaC), including managing your applications, namespaces, keys, queues, rules and more.
+
+"Find it here.":https://github.com/ably/terraform-provider-ably
+
+h2(#github-actions). Ably Control API GitHub Action
+
+A GitHub Action to use the "Ably Control API":https://ably.com/docs/control-api. You can use this action to:
+
+* Create an Ably application.
+* Add an API key to an application, with a list of capabilities.
+
+"Find it here.":https://github.com/ably-labs/ably-control-api-action
+
+h2(#developer-console). Developer console
+
+The "Ably dashboard":ably.com/dashboard contains a developer console. In the developer console you can view connection events. The following screenshot ilustrates an example connection:
+
+<a href="/images/screenshots/developer-console.png" target="_blank">
+  <img src="/images/screenshots/developer-console.png" style="width: 100%" alt="Developer Console">
+</a>
+
+You can attach to a specific channel to view messages. You can also publish messages directly in the console:
+
+<a href="/images/screenshots/developer-console-channels.png" target="_blank">
+  <img src="/images/screenshots/developer-console-channels.png" style="width: 100%" alt="Developer Console - Channels">
+</a>
+
+The developer console provides you with some REST API Curl command snippets that can help you test out Ably quickly:
+
+<a href="/images/screenshots/developer-console-rest-api.png" target="_blank">
+  <img src="/images/screenshots/developer-console-rest-api.png" style="width: 100%" alt="Developer Console - REST API curl samples">
+</a>
+
+Note, for convenience, the Curl commands have your Ably credentials already added.
+
+"Find it here.":ably.com/dashboard
+
+h2(#react-hooks). React hooks
+
+Use Ably in your React application using idiomatic React Hooks.
+
+Using this package you can:
+
+* Interact with Ably channels using a React hook.
+* Send messages via Ably using the channel instances the hooks provide.
+* Get notifications of user presence on channels.
+* Send presence updates.
+
+The hooks provide a simplified syntax for interacting with Ably, and manage the lifecycle of the Ably SDK instances for you taking care to subscribe and unsubscribe to channels and events when your React components re-render.
```

Comment:
First part doesn't flow quite right, one of these feels better to me:

`The hooks provide a simplified syntax for interacting with Ably, and **managing** the lifecycle`
Or
`The hooks provide a simplified syntax **to interact** with Ably, and manage the lifecycle`

Reply:
Rewritten!